frontend/src/components/FormReactHookCreateNewInquiry.js

```diff
@@ -71,7 +71,7 @@ export default function FormReactHookCreateNewInquiry({onAdd}) {
                        ref={register({required: true})}/>
                 {errors.latestDate && "Required"}
 
-                <input type="submit"/>
+                <input className="submitButton" type="submit" value="Post Inquiry!"/>
```

Comment:
you could also use a real button. And you are mixing styled components and css classes